### PR TITLE
ticket allocations and transfers

### PR DESCRIPTION
**Pull request by csell5**

### CODE REVIEW

File: src/dataSources/api.that.tech/orders/mutations.js [hunk 1]
```diff
@@ -72,5 +99,33 @@ export default client => {
...
+    return client
+      .mutation(MUTATION_ALLOCATE_TICKET, variables)
+      .toPromise()
+      .then(({ data, error }) => {
+        if (error) log(error, 'MUTATION_ALLOCATE_TICKET');
+
+        let results;
+
+        if (data) {
+          const { allocateTo } = data.orders.me.order.orderAllocation;
```
Comment (brettski) [inline src/dataSources/api.that.tech/orders/mutations.js:118]: This is an exception if this path doesn't exist. It's not being tested for

**Review by brettski — CHANGES_REQUESTED** [review]
One nit

File: src/dataSources/api.that.tech/orders/mutations.js (hunk 1 above)
Comment (csell5) [reply src/dataSources/api.that.tech/orders/mutations.js:118]: true, but how does that path not ever exist? I guess if I had the order number or allocationid wrong?

File: src/routes/my/components/_TicketAllocation.svelte [hunk 2]
```diff
@@ -0,0 +1,139 @@
+<script context="module">
+  import * as yup from 'yup';
+
+  const schema = yup.object().shape({
+    emailAddress: yup
+      .string()
+      .email()
+      .trim()
+      .required('Please enter your email address.'),
+    orderId: yup.string().required('Please enter your email address.'),
```
Comment (brettski) [inline src/routes/my/components/_TicketAllocation.svelte:10]: Yup checking orderId, though message says email address

File: src/routes/my/components/_TicketAllocation.svelte [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    allocationId: yup.string().required('Please enter your email address.'),
```
Comment (brettski) [inline src/routes/my/components/_TicketAllocation.svelte:11]: Yup checking allocationId, though message says email address

File: src/dataSources/api.that.tech/orders/mutations.js (hunk 1 above)
Comment (brettski) [reply src/dataSources/api.that.tech/orders/mutations.js:118]: Yes, if the API doesn't change that path will remain the same as is.

Comment (csell5) [reply src/dataSources/api.that.tech/orders/mutations.js:118]: This scenario is a problem across all queries and mutations.

File: src/routes/my/components/_TicketAllocation.svelte (hunk 3 above)
Comment (csell5) [reply src/routes/my/components/_TicketAllocation.svelte:11]: I'll fix. It's a hidden field so it never actually shows.

**Review by brettski — APPROVED** [review]

**Pull request merged**